python/cudf/cudf/io/parquet.py

```diff
@@ -64,11 +64,23 @@ def read_parquet(
 
 
 @ioutils.doc_to_parquet()
-def to_parquet(df, path, *args, **kwargs):
+def to_parquet(
+        df,
+        path,
+        compression=None,
+        engine="cudf",
+        *args,
+        **kwargs):
     """{docstring}"""
-    warnings.warn(
-        "Using CPU via PyArrow to write Parquet dataset, this will "
-        "be GPU accelerated in the future"
-    )
-    pa_table = df.to_arrow()
-    pq.write_to_dataset(pa_table, path, *args, **kwargs)
+
+    if engine == "cudf":
+        return libcudf.parquet.write_parquet(
+            cols=df._cols, path=path, compression=compression
+        )
+    else:
+        warnings.warn(
+            "Using CPU via PyArrow to write Parquet dataset, this will "
+            "be GPU accelerated in the future"
+        )
```

Comment:
I think we can safely remove this warning now as we have the GPU accelerated writer :)

Reply:
Should I make this so that the only option is using the GPU accelerated approach then? Or would there be an edge case where the cpu option might be needed by someone?

Reply:
Leave the CPU option, just remove the warning as it's now opt-in